### PR TITLE
remove hostname validation

### PR DESCRIPTION
**Pull request by laipogo**

### CODE REVIEW

File: utils/grpc/client_test.go [hunk 1]
```diff
@@ -72,6 +58,7 @@ func TestParse(t *testing.T) {
 		{in: "http://btfs.io:2333", out: &parsedURL{schema: "http", host: "btfs.io", port: 2333}},
 		{in: "https://127.0.0.1", out: &parsedURL{schema: "https", host: "127.0.0.1", port: 443}},
 		{in: "http://127.0.0.1", out: &parsedURL{schema: "http", host: "127.0.0.1", port: 80}},
+		{in: "hub-service:8080", out: &parsedURL{schema: "http", host: "hub-service", port: 8080}},
```
Comment (thatInfrastructureGuy) [inline utils/grpc/client_test.go]: what is a hub-service:8080 ? Is this related to kubernetes services? @laipogo

Comment (laipogo) [reply utils/grpc/client_test.go]: > what is a hub-service:8080 ? Is this related to kubernetes services? @laipogo

nvm,  it's just a test case. please ignore it.
